cinch/roles/jenkins_master/templates/set_slaveport.groovy

```diff
@@ -0,0 +1,14 @@
+import jenkins.model.Jenkins
+def jenkins = Jenkins.instance
+
+def slave_agent_port = jenkins.getSlaveAgentPort()
+int jenkins_slave_agent_port = {{ jenkins_slave_agent_port|int }}
+
+if (slave_agent_port != jenkins_slave_agent_port) {
+    jenkins.setSlaveAgentPort(jenkins_slave_agent_port)
+    jenkins.save()
+    println "CHANGED: slave agent port from " + slave_agent_port + " to " + \
```

Comment:
I'm not sure the continuation character is necessary, as Groovy is really just Java. But it also shouldn't hurt anything.